Bin/Snapshot.php

```diff
@@ -49,19 +49,34 @@
  */
 class Snapshot extends Console\Dispatcher\Kit
 {
+    /**
+     * User friendly labels for changelog themes.
+     *
+     * @const
+     */
+    public const CHANGELOG_THEMES = [
+        'chore' => 'Quality',
```

Comment:
I think we can keep `Chore`, it's self-explanatory. `Quality` is not always adequate.